configure.ac

```diff
@@ -1,7 +1,7 @@
 dnl Process this file with autoconf to produce a configure script.
 
 AC_PREREQ(2.57)
-AC_INIT([libfabric], [0.0.2], [ofiwg@lists.openfabrics.org])
+AC_INIT([libfabric], [1.0.0-rc1], [ofiwg@lists.openfabrics.org])
```

Comment:
Can you see if ".rc1" works instead?  We kind of want to mark this as the rc1...